Description: Force the local build of libjchart2d-java
  The libjchart2d-java package is not available on the rosdistro
  registry. To workaround the problem, this patch force the
  download and install of the java library.
Origin: https://github.com/RobotLocomotion/ros-drake-vendor/issues/3

diff --git a/tools/workspace/java.bzl b/tools/workspace/java.bzl
index a6b8372772..e412bf1933 100644
--- a/tools/workspace/java.bzl
+++ b/tools/workspace/java.bzl
@@ -14,7 +14,7 @@ def _impl(repo_ctx):
     build_content = """\
 package(default_visibility = ["//visibility:public"])
 """
-    if os_result.target in repo_ctx.attr.local_os_targets:
+    if False:
         is_local = True
         filename = basename(repo_ctx.attr.local_jar)
         repo_ctx.symlink(
